exercises/poker/README.md

```diff
@@ -0,0 +1,47 @@
+# Poker
+
+Pick the best hand(s) from a list of poker hands.
+
+See [wikipedia](https://en.wikipedia.org/wiki/List_of_poker_hands) for an overview of poker hands.
+
+## Hints
+
+- Ranking a list of poker hands can be considered a sorting problem.
```

Comment:
these hints are Rust-specific, therefore for the README generator to not remove them, they need to be placed in .meta/hints.md . I did so in a commit.